spec/shared_examples_prison_slots_and_data.rb

```diff
@@ -0,0 +1,43 @@
+RSpec.shared_examples "slots and data" do
```

Comment:
This is a nice refactoring-out of these :stars:

Might be nice if the name had `prison` in it, though?